src/assets/YAML/default/BuildAndDeployment/Deployment.yaml

```diff
@@ -96,6 +98,12 @@ Build and Deployment:
         iso27001-2022:
           - 5.37
           - 8.32
+      assessment: |
+        - Deployment process is documented and available to relevant staff
+        - All deployment steps are automated and version-controlled
+        - Approvals and access controls are enforced for production deployments
+        - Rollback procedures are defined and tested
```

Comment:
This is an extra activity [Rolling update on deployment](https://dsomm.owasp.org/activity-description?uuid=85d52588-f542-4225-a338-20dc22a5508d&dimension=Build%20and%20Deployment&subDimension=Deployment&level=3&activityName=Rolling%20update%20on%20deployment)